Version 8.1.1 listing nonexistent browser app?

**Describe the bug**
Version 8.1.1 of the app found "qutebrowser" on my system, which was never installed.

**To Reproduce**
Steps to reproduce the behaviour:
1. Click on a link, the window shows qutebrowser along with other browsers I have.
2. Click on qutebrowser, window disappears, nothing happens, because I've never installed that browser.

**Expected behaviour**
That browser should not be in the browser chooser window.

**Screenshots**
<img width="332" alt="image" src="https://user-images.githubusercontent.com/16017904/68270675-16d76f80-002c-11ea-9cb3-2f30327a2930.png">
Three browsers at the top are dimmed because I was holding Alt when taking screenshot.


**Desktop (please complete the following information):**
 - OS: macOS 10.14.6
 - Version 8.1.1

**Additional context**
This problem does not occur on 10.15.1. 
Error messages when running the app binary from the terminal:
```(node:9076) UnhandledPromiseRejectionWarning: Error: Command failed: sh -c open "https://diverse.direct/order/" -b org.qt-project.Qt.QtWebEngineCore
LSGetApplicationForInfo() failed with error -10814 while trying to determine the application with bundle identifier org.qt-project.Qt.QtWebEngineCore.


    at w (/Applications/Browserosaurus.app/Contents/Resources/app.asar/.webpack/main/index.js:1:54859)
    at /Applications/Browserosaurus.app/Contents/Resources/app.asar/.webpack/main/index.js:1:55896
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
(node:9076) UnhandledPromiseRejectionWarning: Error: Command failed: sh -c open "https://diverse.direct/order/" -b org.qt-project.Qt.QtWebEngineCore
LSGetApplicationForInfo() failed with error -10814 while trying to determine the application with bundle identifier org.qt-project.Qt.QtWebEngineCore.


    at w (/Applications/Browserosaurus.app/Contents/Resources/app.asar/.webpack/main/index.js:1:54859)
    at /Applications/Browserosaurus.app/Contents/Resources/app.asar/.webpack/main/index.js:1:55896
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
(node:9076) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:9076) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:9076) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:9076) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

> > One machine is IT maintained and will not move to Catalina for a couple of months until verification is complete.
> 
> Interesting. Maybe it shouldn't have random software installed on it, made by some internet loon 😏
> 
> Seriously though, the app is free, I maintain this in my spare time and cannot afford to keep macs around on old versions of macOS just for regression testing. If you'd like to run the command mentioned above, it will give me more info, but officially the project will only support the version of macOS that I'm currently using 😄

Understood!

I've tried to run the command you suggest, but it didn't work the first time. Before making any attempt at logging, here is what I had to install - am I on the correct lines?

```
$ brew install yarn
$ npm install ts-node
$ npm install tslib
```

I'm in the root of the project:

```
$ pwd
<snip>/browserosaurus/browserosaurus-master
```

Now I run the command:
```
$ yarn list-browsers
yarn run v1.19.1
$ ts-node ./scripts/listInstalledBrowsers

/<snip>/browserosaurus/browserosaurus-master/node_modules/ts-node/src/index.ts:245
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
src/utils/getInstalledBrowsers.ts:1:22 - error TS2307: Cannot find module 'child_process'.

1 import { exec } from 'child_process'
                       ~~~~~~~~~~~~~~~
src/utils/getInstalledBrowsers.ts:2:27 - error TS2307: Cannot find module 'util'.

2 import { promisify } from 'util'
                            ~~~~~~

    at createTSError (/<snip>/browserosaurus/browserosaurus-master/node_modules/ts-node/src/index.ts:245:12)
    at reportTSError (/<snip>/browserosaurus/browserosaurus-master/node_modules/ts-node/src/index.ts:249:19)
    at getOutput (/<snip>/browserosaurus/browserosaurus-master/node_modules/ts-node/src/index.ts:357:34)
    at Object.compile (/<snip>/browserosaurus/browserosaurus-master/node_modules/ts-node/src/index.ts:415:32)
    at Module.m._compile (/<snip>/browserosaurus/browserosaurus-master/node_modules/ts-node/src/index.ts:493:43)
    at Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Object.require.extensions.<computed> [as .ts] (/<snip>/browserosaurus/browserosaurus-master/node_modules/ts-node/src/index.ts:496:12)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.```

--- a/src/utils/getInstalledBrowsers.ts
+++ b/src/utils/getInstalledBrowsers.ts
@@ -25,7 +25,7 @@ export const getInstalledBrowsers = (): Promise<BrowserName[]> =>
   filterAsync(browserNames, async (name: BrowserName) => {
     const { appId } = browsers[name]
     const { stdout: appPath } = await execP(
-      `mdfind kMDItemCFBundleIdentifier = "${appId}"`,
+      `mdfind kMDItemCFBundleIdentifier = "${appId}" | grep -v '.framework$' || true`,
     )
     return !!appPath
   })

--- a/src/utils/getInstalledBrowsers.ts
+++ b/src/utils/getInstalledBrowsers.ts
@@ -25,7 +25,7 @@ export const getInstalledBrowsers = (): Promise<BrowserName[]> =>
   filterAsync(browserNames, async (name: BrowserName) => {
     const { appId } = browsers[name]
     const { stdout: appPath } = await execP(
-      `mdfind 'kMDItemContentType == "com.apple.application-bundle" && kMDItemCFBundleIdentifier = "${appId}"'`,
+      `mdfind 'kMDItemContentType == "com.apple.application-bundle" && kMDItemCFBundleIdentifier == "${appId}"'`,
     )
     return !!appPath
   })

--- a/src/utils/getInstalledBrowsers.ts
+++ b/src/utils/getInstalledBrowsers.ts
@@ -25,7 +25,7 @@ export const getInstalledBrowsers = (): Promise<BrowserName[]> =>
   filterAsync(browserNames, async (name: BrowserName) => {
     const { appId } = browsers[name]
     const { stdout: appPath } = await execP(
-      `mdfind kMDItemCFBundleIdentifier = "${appId}" | grep -v '.framework$' || true`,
+      `mdfind 'kMDItemContentType == "com.apple.application-bundle" && kMDItemCFBundleIdentifier = "${appId}"'`,
     )
     return !!appPath
   })

